henryade/Banka-React
# #168078505 Implement Sign Up Feature

#### What does this PR do?
Implement sign up feature

#### Description of Task to be completed?
- add components and styles
- add actions, action types, and reducers
- add custom functions

#### How should this be manually tested?
Open [Link](https://banka-app-in-react.herokuapp.com/signup) to see page.
Input your details and click register

#### Any background context you want to provide?
 - N/A
#### What are the relevant pivotal tracker stories?
[#168078505](https://www.pivotaltracker.com/story/show/168078505)
#### Screenshots (if appropriate)
![Screenshot from 2019-11-13 16-36-21](https://user-images.githubusercontent.com/47576037/68778695-ccf61700-0633-11ea-8cf0-6a9200d71959.png)
